extensions/schema-compare/src/test/testSchemaCompareMainWindow.ts

```diff
@@ -69,6 +69,19 @@ export class SchemaCompareMainWindowTest extends SchemaCompareMainWindow {
 			result = true;
 		}
 
+		if (result === false) {
```

Comment:
Why not just remove the initial if and then use should to actually do the verification here?

Reply:
Or just return true immediately, leaving the logging if it never returns.

Reply:
I am keeping the actual verification to the test cases only. 
Updated it based on Benjin's comment.